### ISSUE TITLE
DynamoDB on-demand capacity mode support

### ISSUE THREAD

**Issue by alanmm9** [localstack/localstack#1089]
When I am trying to create a DynamoDB table using AWS Java SDK and I set `withBillingMode(PAY_PER_REQUEST)`, I get an exception: 
```
Caused by: com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException: No provisioned throughput specified for the table (Service: AmazonDynamoDBv2; Status Code: 400; Error Code: ValidationException; Request ID: f5b449d5-f6d2-4ade-b249-78e7950bfd01)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1695)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1350)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1101)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:758)
```

Are you planning to support this in the future?

**Comment by lwoodson** [comment]
Slightly different but related issue here: https://github.com/localstack/localstack/issues/1193

**Comment by lwoodson** [comment]
Also, the issue referenced in this bug report seems resolved for me using localstack `v0.9.0` in a docker container.  I can create and use a table using `PAY_PER_REQUEST` billing and use it as expected, but not if the cloudformation definition for the table includes `GlobalSecondaryIndexes`.

**Comment by whummer** [comment]
This issue should be fixed in the latest version. Tracking the issue around PAY_PER_REQUEST in combination with GSIs in this ticket: #1737 Thanks
